core/src/com/unciv/logic/city/CityReligion.kt

```diff
@@ -16,18 +14,32 @@ class CityInfoReligionManager {
     val religionsAtSomePointAdopted: HashSet<String> = hashSetOf()
 
     private val pressures: Counter<String> = Counter()
+    // `getNumberOfFollowers()` was called a surprisingly large amount of time, so caching it feels useful
+    private val followers: Counter<String> = Counter()
```

Comment:
If this is a cached value, make it a @Transient

Reply:
Sometimes just "reading" a save in notepadqq can give hints. We already got some stuff in there that could be avoided...